salt/docker_apps/files/credere.yaml

```diff
@@ -1,8 +1,18 @@
-# Placeholder for custom application
-version: "3"
+x-shared: &shared
+  env_file: .env
+  user: "{{ pillar.docker.uid }}:{{ pillar.docker.uid }}"
+  extra_hosts:
+    - "host.docker.internal:host-gateway"
+  restart: unless-stopped
+
 services:
```

Comment:
The services don't need any volumes for uploaded files? I don't remember how those are handled.

pelican-frontend does something like: https://github.com/open-contracting/pelican-frontend/blob/main/backend/core/settings.py#L223-L233